test/parallel/test-fs-chmod.js

```diff
@@ -139,5 +140,11 @@ if (fs.lchmod) {
 
 
 process.on('exit', function() {
+  try {
```

Comment:
Is this added because the files being operated on are in the `fixtures` dir? IMO, the test should be refactored to use the tmp dir instead so that we don't need to do this. Tests shouldn't modify files in fixtures. Then it can check mode on file creation too, I suppose.